code/modules/research/xenobiology/crossbreeding/gentle.dm

```diff
@@ -28,27 +28,19 @@
 	. = ..()
 	STOP_PROCESSING(SSobj,src)
 
-/obj/item/slimecross/gentle/attack_self(mob/user)
-	if(cooldown > 0)
+/obj/item/slimecross/gentle/attack_self(mob/living/carbon/user)
+	if(cooldown > 0 || user.incapacitated())
```

Comment:
You don't validate that `user` is a carbon, this could runtime.

Reply:
I mean, how would a non carbon trigger `attack_self`?

Reply:
I'd be happily willing to bet that there's something that isn't a carbon that can `attack_self`. Just put in an `!istype(user)` check, those are O(1) now anyways